### PR TITLE
Integration tests for /service/start

### PR DESCRIPTION
**Pull request by cruhland**
This depends on #551. Please review/merge that first!

### CODE REVIEW

**Comment by cruhland** [comment]
@jsancio @larioj ready for review.

File: cosmos-server/src/it/scala/com/mesosphere/cosmos/ServiceStartSpec.scala [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+      val typedResponse = decode[rpc.v1.model.ServiceStartResponse](startResponse.contentString)
+      assertResult(packageName)(typedResponse.packageName)
+
+      val Some(appId) = typedResponse.appId
+      assertResult("/cassandra")(appId.toString)
+
+      val marathonApp = getMarathonApp(appId)
+      assertResult(appId)(marathonApp.id)
+    }
```
Comment (jsancio) [inline cosmos-server/src/it/scala/com/mesosphere/cosmos/ServiceStartSpec.scala]: Can we test "successfully running a package that was installed from a DC/OS package (package.dcos)"?

**Comment by jsancio** [comment]
LGTM. Minor comment.

**Comment by cruhland** [comment]
@jsancio updated.

File: cosmos-server/src/it/scala/com/mesosphere/cosmos/ServiceStartSpec.scala [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  import ServiceStartSpec._
+
+  after {
+    val uninstallCassandra =
+      rpc.v1.model.UninstallRequest("cassandra", appId = None, all = Some(true))
+    val uninstallHelloworld =
+      rpc.v1.model.UninstallRequest("helloworld", appId = None, all = Some(true))
+
+    // Enclose in braces so we can ignore the return values
+    { val _ = CosmosClient.submit(CosmosRequests.packageUninstall(uninstallCassandra)) }
```
Comment (larioj) [inline cosmos-server/src/it/scala/com/mesosphere/cosmos/ServiceStartSpec.scala]: I'm fairly sure you only need to ignore the return value for the last statement in the block.

**Comment by cruhland** [comment]
@larioj @jsancio back to you

**Pull request merged**